curriculum/challenges/english/08-coding-interview-prep/rosetta-code/zhang-suen-thinning-algorithm.english.md

```diff
@@ -7,7 +7,7 @@ challengeType: 5
 ## Description
 <section id='description'>
 This is an algorithm used to thin a black and white i.e. one bit per pixel images.
-For example, with an input image of:
```

Comment:
Could we change this back to "with"? The word "for" works fine, but is a bit repetitive since the sentence starts with the same word. Also, I'd like to keep the language as close to the original as possible.